lib/ansible/modules/remote_management/redfish/redfish_info.py

```diff
@@ -206,6 +206,14 @@
       username: "{{ username }}"
       password: "{{ password }}"
 
+  - name: Get Manager Services
+    redfish_command:
+      category: Manager
+      command: GetManagerServices
```

Comment:
```suggestion
      command: GetNetworkProtocols
```

See suggestion in redfish_config about naming.

Reply:
Done.